l10n_eu_product_adr_dangerous_goods/models/product_product.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    # To be set manually for the moment. Could eventually be computed after,
+    # depending on the dangerous class and the product's weight/captain'sage…
+    limited_quantity = fields.Boolean()
```

Comment:
I suppose migration scripts will come later as these fields were in the `l10n_eu_product_adr` addon in 13.0

Reply:
Answer: migration script has been done there: https://github.com/StefanRijnhart/community-data-files/pull/1